repository/spark/operator/params.yaml

```diff
@@ -114,6 +114,11 @@ parameters:
     default: ""
     displayName: "Spark History Server Persistent Volume Claim Name"
 
+  - name: historyServerSparkConfSecret
+    description: "Name of a Secret with Spark config file. The secret should be in the same namespace as an Operator."
+    default: ""
+    displayName: "Name of a Secret with AWS credentials"
```

Comment:
This looks a little bit misleading because (at least from my understanding) this secret contains the full config for History server,  isn't it? There's another way to configure it using `historyServerOpts` so it looks like `historyServerSparkConfSecret` should be used only for sensitive data.